css/style.css

```diff
@@ -101,7 +103,9 @@ footer {
 }
 
 .testimonial {
-  color: #f0efef
+  color: #f0efef;
+  background: url(../images/testimonial_bg.jpg) no-repeat center top rgb(134,141,157);
```

Comment:
@sofiaayala I would move this to the header, since this image is only in the header. It would also give you more control to position it. Try to get the whole image within the header 🔨